base/strings/util.jl

```diff
@@ -85,15 +85,15 @@ julia> a = "March"
 julia> chop(a)
 "Marc"
 
-julia> chop(a, 1, 2)
+julia> chop(a, head = 1, tail = 2)
 "ar"
 
-julia> chop(a, 5, 5)
+julia> chop(a, head = 5, tail = 5)
 ""
 ```
 """
 chop(s::AbstractString) = SubString(s, start(s), prevind(s, endof(s)))
-chop(s::AbstractString, head::Integer, tail::Integer) =
+chop(s::AbstractString; head::Integer = 0, tail::Integer = 1) =
```

Comment:
This will overwrite the method on the line above, right?

Reply:
Oh shoot I messed this up. If I understand correctly `start(s)` and `prevind(s, endof(s))` are supposed to be optimizations for the default cause. If I'm understanding correctly constant propagation is not working right now for keyword arguments. So maybe I should comment out the first method and add back in the optimization if that ever happens?